src/riot-rs-threads/src/lib.rs

```diff
@@ -49,17 +52,41 @@ pub mod macro_reexports {
 pub use riot_rs_runqueue::{RunqueueId, ThreadId};
 pub use thread_flags as flags;
 
+#[cfg(feature = "core-affinity")]
+pub use smp::CoreAffinity;
+#[cfg(feature = "multi-core")]
+pub use smp::CoreId;
```

Comment:
Can we have a `CoreId` also on single-core? and `riot_rs_threads::core_id()`? (always returning 0).
(can do as follow up)

Reply:
Done